paasta_tools/cli/cmds/boost.py

```diff
@@ -102,21 +97,23 @@ def paasta_boost(args):
     soa_dir = args.soa_dir
     system_paasta_config = load_system_paasta_config()
     all_clusters = list_clusters(soa_dir=soa_dir)
-    if args.cluster in all_clusters:
-        print("Valid cluster")
-        return_code, output = execute_paasta_cluster_boost_on_remote_master(
-            cluster=args.cluster,
-            system_paasta_config=system_paasta_config,
-            action=args.action,
-            region=args.region,
-            pool=args.pool,
-            duration=args.duration if args.action == 'set' else None,
-            override=args.override if args.action == 'set' else None,
-            boost=args.boost if args.action == 'set' else None,
-            verbose=args.verbose,
-        )
-    else:
-        paasta_print("Cluster %s doesn't look like a valid cluster" % args.cluster)
-        paasta_print("Here is a list of valid paasta clusters:\n{}".format(all_clusters))
-        return_code = 1
+    for cluster in args.cluster:
+        if cluster not in all_clusters:
```

Comment:
I thought we had a function to handle validating a cluster, but I'm struggling to find it right now. Can you either find it or make one (somewhere more generic than in this file)?

Reply:
Are you thinking about this one ? https://github.com/Yelp/paasta/blob/master/paasta_tools/cli/cmds/metastatus.py#L120